### DISCUSSION TITLE
How do I rotate the erlang cookie on a production cluster?

### DISCUSSION THREAD

**Discussion by dmitry-okun** [rabbitmq/rabbitmq-server#9784] [Questions]
I am looking to rotate the erlang cookie without down time. I cant find anywhere in the documentation on how to do it. Has anyone done it before?

**Comment by lukebakken** [comment]
https://www.erlang.org/doc/man/erlang#set_cookie-2

```
rabbitmqctl eval "erlang:set_cookie('NEWCOOKIEATOM')."
```

You would have to also ensure that the local cookie file has the updated value. Note that I have never tried this so you're on your own!
